this seems to check whether the test driver should announce failures to some
irc channel. just don't.

diff -ruN a/test/irc.lua b/test/irc.lua
--- a/test/irc.lua	2024-08-25 03:34:54.000000000 +0200
+++ b/test/irc.lua	2024-08-26 17:44:13.886052479 +0200
@@ -1,17 +1,7 @@
-local server_port = (os.getenv("IRC_HOST_PORT") or "irc.libera.chat 6667")
-local channel = os.getenv("IRC_CHANNEL")
-local url = os.getenv("JOB_URL") or "???"
-
-local remote = io.popen("git remote get-url origin"):read('*l')
-local is_origin = remote:find('~technomancy/fennel$') ~= nil
-
-local branch = io.popen("git rev-parse --abbrev-ref HEAD"):read('*l')
-local is_main = branch == 'main'
-
 -- This may fail in future if libera chat once again blocks builds.sr.ht
 -- from connecting; it currently works after we asked them to look into it
 return function(failure_count)
-    if  (0 ~= tonumber(failure_count)) and is_main and is_origin and channel then
+    if false then
         print("Announcing failure on", server_port, channel)
 
         local git_log = io.popen("git log --oneline -n 1 HEAD")
